_pages/ontology.md

```diff
@@ -0,0 +1,100 @@
+---
+permalink: /ontology/
+title: CSL Ontology
+description: Components of the CSL data model can be referenced with unique identifiers
+redirect_from:
+  # categories
+  - /ontology/category/
+  - /ontology/category/anthropology
+  - /ontology/category/astronomy
+  - /ontology/category/biology
+  - /ontology/category/botany
+  - /ontology/category/chemistry
+  - /ontology/category/communications
+  - /ontology/category/engineering
+  - /ontology/category/generic-base
+  - /ontology/category/geography
+  - /ontology/category/geology
+  - /ontology/category/history
+  - /ontology/category/humanities
+  - /ontology/category/law
+  - /ontology/category/linguistics
+  - /ontology/category/literature
+  - /ontology/category/math
+  - /ontology/category/medicine
+  - /ontology/category/philosophy
+  - /ontology/category/physics
+  - /ontology/category/political_science
+  - /ontology/category/psychology
+  - /ontology/category/science
+  - /ontology/category/social_science
+  - /ontology/category/sociology
+  - /ontology/category/theology
+  - /ontology/category/zoology
+  # roles
+  - /ontology/role/
+  - /ontology/role/author
+  - /ontology/role/collection-editor
+  - /ontology/role/composer
+  - /ontology/role/container-author
+  - /ontology/role/director
+  - /ontology/role/editor
+  - /ontology/role/editorial-director
+  - /ontology/role/editortranslator
+  - /ontology/role/illustrator
+  - /ontology/role/interviewer
+  - /ontology/role/original-author
+  - /ontology/role/recipient
+  - /ontology/role/reviewed-author
+  - /ontology/role/translator
+  # types
+  - /ontology/type/article
```

Comment:
What about the `/ontology/type/` as you also have `/ontology/role/` and `/ontology/category/`? Or maybe we don't need any of these?
```suggestion
  - /ontology/type/
  - /ontology/type/article
```